languages/python/oso/tests/test_data_filtering.py

```diff
@@ -293,6 +319,22 @@ def test_no_relationships(oso, t):
     assert len(results) == 3
 
 
+def test_neq(oso, t):
```

Comment:
What's the approach on testing been?

Are we writing tests that confirm that the core produces the right filter plan from a policy in the core? Or are those all in Python?

If they're in Python how are we keeping track of which tests need to be ported to all languages and which only need to be in Python / one language?